### ISSUE TITLE
Handling more than one output, but only one loss

### ISSUE THREAD

**Issue by jonahweissman** [stared/livelossplot#10]
I have a model that has three outputs (named "usage", "clients", and "sessions") and one loss function (mean absolute percentage error.) As shown in the picture below, the names of my plots are a little messed up. I believe the upper right plot should be titled "usage_loss".
![image](https://user-images.githubusercontent.com/19804455/38181849-5c875b0c-3603-11e8-9d44-dbabc9719ea0.png)

I think the problem is in this section of the code, but I'm not sure how to fix it.

https://github.com/stared/livelossplot/blob/fd72c53bd82d0c0052478c735edd356892f6f7f6/livelossplot/keras_plot.py#L54-L59

**Comment by stared** [comment]
@jonahweissman I admit I didn't test `livelossplot` with multiple losses. I wrote based on my understanding on how does Keras deal with multiple losses. 

Could you show me the line with `model.compile` and the result of `model.loss`, `model.output_names` (and possibly, what is in `logs`)?

**Comment by jonahweissman** [comment]
Here's my line with `model.compile` (I'm using the functional API):
```python
model = Model(inputs=[weather_layers[0], day_layers[0], date_layers[0]], outputs=[usage_out,clients_out,sessions_out])
model.compile(loss='mean_absolute_percentage_error',
              optimizer=Adam(0.0001))
```
```python
In [1]: model.loss
Out[1]: 'mean_absolute_percentage_error'

In [2]: model.output_names
Out[2]: ['usage', 'clients', 'sessions']

In [3]: model.losses
Out[3]: []
```

I didn't specify loss functions for each output, so I believe it uses `model.loss` for each.

**Comment by stared** [comment]
I see. Apparently I would need to loop through output names, in addition to losses (though: are many losses a typical thing?).

To do the whole thing - https://github.com/keras-team/keras/blob/master/keras/engine/training.py is a bit convolved. 

If you want to give it a try, you are welcome (feel free to change this piece of code a lot - as I said, didn't test it so don't be afraid of a complete rewrite of this part).
